src/main/kotlin/app/revanced/patches/messenger/compose/patch/DisableComposeExpressionTabChange.kt

```diff
@@ -0,0 +1,36 @@
+package app.revanced.patches.messenger.compose.patch
+
+import app.revanced.extensions.toErrorResult
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Package
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.instruction
+import app.revanced.patcher.extensions.replaceInstruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.messenger.compose.fingerprints.SwitchComposeButtonFingerprint
+import org.jf.dexlib2.iface.instruction.OneRegisterInstruction
+
+@Patch
+@Name("disable-compose-tab-switch")
+@Description("Disables switching to sticker search mode in compose field")
+@Compatibility([Package("com.facebook.orca")])
+@Version("0.0.1")
+class DisableComposeExpressionTabChange : BytecodePatch(
```

Comment:
Is it possible to use a better name than "Compose" like "Inputbox" or similar? It might not be clear what "Compose" refers to.

Reply:
Yeah, I agree, changed it now